src/bindings/python/tests/test_graph/test_create_op.py

```diff
@@ -2199,3 +2199,35 @@ def test_interpolate_opset10(dtype, expected_shape, shape_calculation_mode):
     assert node.get_type_name() == "Interpolate"
     assert node.get_output_size() == 1
     assert list(node.get_output_shape(0)) == expected_shape
+
+
+def test_is_inf_opset10_default():
+    input_shape = [2, 2, 2, 2]
+    input_node = ov.parameter(input_shape, dtype=np.float, name="InputData")
+    node = ov_opset10.is_inf(input_node, {})
+
+    assert node.get_type_name() == "IsInf"
+    assert node.get_output_size() == 1
+    assert list(node.get_output_shape(0)) == input_shape
+
+    node_attributes = node.get_attributes()
+    assert node_attributes["detect_positive"] is True
+    assert node_attributes["detect_negative"] is True
+
+
+def test_is_inf_opset10_custom_attributes():
+    input_shape = [2, 2, 2]
+    input_node = ov.parameter(input_shape, dtype=np.float, name="InputData")
+    attributes = {
+        "detect_negative": False,
+        "detect_positive": True,
+    }
```

Comment:
Is it possible to specify only one attribute? It's worth to add such test.
For example with 
    `attributes = {"detect_positive": False}`

Reply:
Done